contribute/issue-template.md

```diff
@@ -0,0 +1,50 @@
+# 🐛 Report for Ada-url
+
+### Description
+
+If you notice something that isn't working as expected or if you have an idea to improve our project, we'd be happy to hear from you! Please report issues, request additional features, documentation, translations, and provide suggestions for improvement. Your help is crucial to continue refining our project and providing the best possible experience for our users. Thank you for your collaboration!
+
+### Type of Report
+
+Please select the type of report below:
+
+- [ ] Bug Report
+- [ ] Feature Request
+- [ ] Other
+
+### Steps to Reproduce
+
+Please provide a step-by-step guide to reproduce the problem, if applicable.
+
+1. ...
+2. ...
+3. ...
+
+### Expected Behavior
+
+Please describe the expected behavior.
+
+### Current Behavior
+
+Please describe the current behavior.
+
+### Screenshots
+
+If possible, attach screenshots that may help clarify the problem.
```

Comment:
I don't think screenshots should be included in the bug report.

Reply:
Removed!